src/Surfnet/ServiceProviderDashboard/Application/Metadata/JsonGenerator.php

```diff
@@ -35,6 +35,10 @@
  */
 class JsonGenerator implements GeneratorInterface
 {
+    public const DEFAULT_CREATED_REVISION_NOTE = 'Entity created';
+    public const DEFAULT_CHANGED_REVISION_NOTE = 'Entity changed';
+    public const DEFAULT_CHANGE_REQUEST_NOTE = 'Change request';
```

Comment:
I'm fine with a hard coded text, but maybe the PM's expect them to be translatable? Personally I always try to make them translatable, and revert them to hard coded when explicitly required.